assets/app/scripts/filters/capitalize.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+angular.module('openshiftConsole')
+  .filter('capitalize', function() {
+    return function(input) {
+      return _.capitalize(input);
+    };
+  });
```

Comment:
We've typically put filters like these in util.js. I'm trying to decide how I feel about having a file for each filter.

It might make sense to have a file for string operations or maybe just case operations like sentenceCase, startCase, etc. (sentenceCase filter is incorrectly in resources.js now.)